internal/ingress/controller/controller.go

```diff
@@ -938,8 +938,13 @@ func (n *NGINXController) createServers(data []*ingress.Ingress,
 		// default upstream name
 		un := du.Name
 
+		if anns.Canary.Enabled {
+			glog.Infof("Ingress %v with backend %v is not eligible to use the catch-all server. skip", ingKey, un)
```

Comment:
@wayt I don't think at this point we can definitely say what's the configured backend for Ingress. Same for below. It will potentially confuse users.

Also IMO we should log this via V(2) or something. Otherwise it'll just spam the log without any value.